docs/_tec-events/2023-08-07-swcsf-meeting.md

```diff
@@ -0,0 +1,105 @@
+- Haaretz, “[What Did the FBI Really Want NSO’s Pegasus For?](https://www.haaretz.com/israel-news/tech-news/2022-02-06/ty-article/.premium/what-did-the-fbi-really-want-nsos-pegasus-for/0000017f-e8d5-df2c-a1ff-fed5ef530000)” Feb. 6, 2022
+- Amnesty.org, “[The Pegasus Project: One year on, spyware crisis continues after failure to clamp down on surveillance industry](https://www.amnesty.org/en/latest/news/2022/07/the-pegasus-project-one-year-on-spyware-crisis-continues-after-failure-to-clamp-down-on-surveillance-industry/),” July 18, 2022
+- Council on Foreign Relations, [cfr.org](https://www.cfr.org), “[How Israel’s Pegasus Spyware Stoked the Surveillance Debate](https://www.cfr.org/in-brief/how-israels-pegasus-spyware-stoked-surveillance-debate),” March 8, 2022
+- [The Pegasus Project](https://www.theguardian.com/news/series/pegasus-project), The Guardian, ongoing.
+- New York Times, “[A Front Company and a Fake Identity: How the U.S. Came to Use Spyware It Was Trying to Kill](https://www.nytimes.com/2023/04/02/us/politics/nso-contract-us-spy.html),” April 2, 2023 (updated April 10)
+- ACLU Newsletter, “[Five Things to Know About NSA Mass Surveillance and the Coming Fight in Congress](https://www.aclu.org/news/national-security/five-things-to-know-about-nsa-mass-surveillance-and-the-coming-fight-in-congress#:~:text=FBI%20agents%20routinely%20run%20searches,to%20do%20with%20national%20security.),” April 11, 2023
+- New York Times, “[Who Paid for a Mysterious Spy Tool? The F.B.I., an F.B.I. Inquiry Found](https://www.nytimes.com/2023/07/31/us/politics/nso-spy-tool-landmark-fbi.html),” July 31, 2023
+
+## Topic 2: Android Privacy Tips
+
+While the fight against military grade spyware is long and slow, there are some relatively simple steps that can be taken to improve your privacy and harden your Android phone against more modest threats.
+
+Why aren’t we covering Apple? Well, Apple does not allow modification of their devices via either hardware or software. If you have an Apple device, you are limited to the hardening and security mitigations that they have provided.
+
+We’ll cover what you can protect and the limits of said protection for:
+- network traffic
+- location data
+- eavesdropping (visual and auditory)
+- Android permissions
+- permissions that are more dangerous than they seem e.g. the accelerometer
+- a few side channels
+
+As well as more permanent solutions:
+- physical blocking
+- privacy front-ends
+- chaffing (data pools)
+- open source hardware and software to prevent or slow enshittification
+- legislative solutions and their limits
+
+## Location
+
+For this meeting, we are back in Room SO1330 of the South Building at GateWay Community College, Washington Campus. Details below.
+Schedule
+
+### Attend Virtually
+
+Zoom meeting starts at: 6:15 PM MST (Arizona time)
+
+Join the Zoom Meeting via computer
+
+Meeting ID: 944 8607 3474
+
+Passcode: 292977
```

Comment:
Should we stylize the meeting invite block to make it stand out?

Reply:
We can visit in a future PR.